packages/cli/lib/usageTracking.js

```diff
@@ -132,10 +151,48 @@ const trackAuthAction = async (command, authType, step, accountId) => {
   }
 };
 
+function trackCommandMetadataUsage(command, meta = {}, accountId) {
```

Comment:
Elected to create a new function rather than adjusting `trackingCommandUsage`, but if you feel its easier to re-use that one let me know

Reply:
I like this 👌